example.json

```diff
@@ -32,7 +32,8 @@
     "linkText": "View",
     "duration": "2 mins",
     "_pageLevelProgress": {
-        "_isEnabled": true
+        "_isEnabled": true,
+        "_excludeAssessments": true
```

Comment:
can you add missing schema properties to the example? `_showPageCompletion` i think might be the only one?

Reply:
_excludeAssessments default value in schema and example isn't consistent, does it make intentionally ? apart from that all looks good